hannah-hjudge/Homework-3/json-homework/json-hw.js

```diff
@@ -0,0 +1,55 @@
+
+/*
+1. Convert the string `jsonString` to a Javascript Object and store it in a variable
+called `flickrObj`
+*/
+
+let flickrObj = JSON.parse(jsonString);
+
+
+/*
+2. Iterate (this means 'use a loop') and do a console.log of each of the item titles
+with the date the photo was taken. The first console.log should return the following:
+
+"Edelweiss Air, Airbus A320-214, HB-IJU 2016-02-23T18:48:14Z"
+
+*/
+
+// this was my attempt 
+// i also tried forEach but i didn't succeed, and then i deleted that code :(
+
+for (let items in flickrObj) {
+	console.log(flickrObj[items].title + flickrObj[items].date_taken);
```

Comment:
Good try! When you use `for/in`, the variable you define (here `items`) is the keyword you use in each iteration of the loop to reference the current element. So rather than `flickrObj[items].title` you should be able to say simply `items.title`. I suggest trying this out and letting me know if you have questions.